scripts/kconfig/hardened.csv

```diff
@@ -73,6 +73,7 @@ BT_OTS_CLIENT,n,experimental
 BT_PASSKEY_KEYPRESS,n,experimental
 BT_PER_ADV_RSP,n,experimental
 BT_PER_ADV_SYNC_RSP,n,experimental
+BT_PNP,n,experimental
```

Comment:
updating this file won't be needed if you rebase to pick up a recent change implemented in the hardening tool. Thanks for the attention to this though!